app/controllers/authentication_controller.rb

```diff
@@ -25,6 +25,19 @@ def clear_extra_publisher_session_entries
   end
 
   def trigger_jobseeker_sign_in_event(success_or_failure, errors = nil)
+    event = DfE::Analytics::Event.new
+      .with_type(:jobseeker_sign_in_attempt)
+      .with_request_details(request)
+      .with_response_details(response)
+      .with_user(current_jobseeker)
+      .with_data(
+        email_identifier: StringAnonymiser.new(params.dig(:jobseeker, :email)),
```

Comment:
Now that the DfE Analytics GEM is being used, if the email is in analytics_pii.yml, then we should use the DfE:Analytics anonymiser ie:
```Ruby
DfE::Analytics.anonymise(params.dig(:jobseeker, :email)) 
```

Otherwise the GEM anonymiser is likely to use a different hashing algorithm from a custom anonymiser, and joining up this value from database tables will not work as expected.

This is a general comment that is applicable throughout this PR.